modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/mdf-claim-manager-status/index.html

```diff
@@ -266,10 +266,10 @@
 				</li>
 			[/#if]
 
-			[#if hasClaimPaidAction]
+			[#if hasApproveAction]
 				[#assign count = count + 1]
-				<li class="dropdown-item" id="dropdownMDFClaimStatusItem" onclick="callClaimPaidModal('claimPaid')">
-					Claim Paid
+				<li class="dropdown-item" id="dropdownMDFClaimStatusItem" onclick="callMDFClaimModal('approved')">
+					Approved
```

Comment:
Hy @netosantosbr 
Good work here.
I think this wouldn't need to be here as we talked about.
Approved is not part of the Request More Info flow.

Reply:
Thank you for reviewing, @caiosouza15.
I sent the requested changes about this.